main.py

```diff
@@ -0,0 +1,124 @@
+import requests
+import re
+from lxml import html
+from lxml.cssselect import CSSSelector
+import csv
+
+urlinit = 'http://www.schooldays.ie'
+
+def regionurl():
+
+#Collect the content of each schooldays.ie urls for region Dublin-1, Dublin-2 ....Dublin-18 and calls schollurl() function,
+#Create the CSV file with the header
+
+    header = ['School Name', 'Address', 'Postal District', 'Year', 'Total Sitting Leaving', #csv reader
+          'Total Main Colleges']
+    with open('./schoolfile', 'a') as myfile: #write the csv reader
+        wr = csv.writer(myfile, quoting=csv.QUOTE_ALL)
+        wr.writerow(header)
+
+    dublinregions = list(range(1,19))
+    otherdublinregions = ['6w', '20', '22', '24', 'north-county-dublin', 'south-county-dublin', 'west-county-dublin']
```

Comment:
recommended `maximum line length` according to [PEP8](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) it's 79 characters. It isn't a rule.. but remember that one of the most beautiful thing about python is the code readability.

Reply:
ok